src/pages/Home/components/CountriesHeader/style.scss

```
@@ -0,0 +1,60 @@
+.section__countries > div {
+  margin-top: 82px;
+}
+
+.section__countries,
+.section__leagues {
+  margin: 38px 0;
+
+  display: flex;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)